shell/src/main/java/alluxio/cli/fs/command/CpCommand.java

```diff
@@ -793,16 +803,16 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
   @Override
   public String getUsage() {
     return "cp "
-        + "[-R] "
+        + "[-R -r --recursive] "
```

Comment:
can we support same set of options, -R and --recursive should be enough
or we can research to see whether -r is a also a commonly used one

Reply:
If we are consistent with the linux manual, according to https://man7.org/linux/man-pages/man1/cp.1.html, we need to support -r. But if -r is not supported, we can delete RECURSIVE_ALIAS_OPTION, and the code will be more elegant.